Dockerfile

```diff
@@ -41,5 +42,17 @@ WORKDIR $APP_HOME
 RUN chown node:node . 
 USER node
 
+# stage 2 
+# make image "Distroless" to contain only the app and its runtime dependencies
+FROM gcr.io/distroless/nodejs18-debian11
```

Comment:
maybe would wish to understand what OS the virtual machine runs on, if its debian...